### PR TITLE
Shambhawi

### PR DESCRIPTION
**Pull request by shambhawiprakash**

### CODE REVIEW

File: pages/services.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
```
Comment (ankitc1010) [inline pages/services.js]: This should be there in `components/pages/Rental/index.js`.

Comment (shambhawiprakash) [reply pages/services.js]: My assignment is for service and repair, not rental. So should it be 
components/pages/ServicesRepair/index.js  ?

Comment (ankitc1010) [reply pages/services.js]: Yeap.

File: pages/services.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// please follow the directory structure of contact.js
```
Comment (shambhawiprakash) [inline pages/services.js:1]: Should I delete the `comments`?

Comment (ankitc1010) [reply pages/services.js:1]: No let it stay there.

**Pull request merged**